hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyCreateRequest.java

```diff
@@ -806,7 +807,7 @@ private void checkNotAValidPath(String keyName) {
       omKeyCreateRequest.preExecute(ozoneManager);
       fail("checkNotAValidPath failed for path" + keyName);
     } catch (IOException ex) {
-      assertTrue(ex instanceof OMException);
+      assertInstanceOf(OMException.class, ex);
       OMException omException = (OMException) ex;
```

Comment:
nit: cast can be removed

```suggestion
      OMException omException = assertInstanceOf(OMException.class, ex);
```